web/router.go

```diff
@@ -121,6 +120,9 @@ func v2Routes(app *services.ChainlinkApplication, engine *gin.Engine) {
 		authv2.GET("/bridge_types/:BridgeName", bt.Show)
 		authv2.DELETE("/bridge_types/:BridgeName", bt.Destroy)
 
+		w := WithdrawController{app}
+		authv2.POST("/withdraw", w.Create)
```

Comment:
optional nitpick: `withdraw` is a verb, `withdrawal` is a noun. Seems better suited as a resource and controller name. Withdraw creates a Withdrawal, with `withdraw` being the POST.

`POST` creates a `withdrawal` at `/withdrawals`